### PR TITLE
{system}[GCCcore/11.3.0] file v5.43

### PR DESCRIPTION
**Pull request by GrahamDB**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'file'
+version = '5.41'
```
Comment (Micket) [inline easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb:4]: Looks like 5.43 is out so lets update it

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb (hunk 1 above)
Comment (GrahamDB) [reply easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb:4]: Added easyconfig file for file-5.43-GCCcore-11.3.0.eb; did you want it submitted as a separate pr?

Comment (Micket) [reply easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb:4]: well , actually i wanted *only* that version. I see no reason for the 5.41 GCCcore-11.3.0 version to exist?

**Comment by GrahamDB** [comment]
So should I cancel this pull request and start a new one, or do I did need
to figure how to delete file-5.41-GCCcore-11.3.0.eb from this pull request?

**Comment by Micket** [comment]
> So should I cancel this pull request and start a new one, or do I did need to figure how to delete file-5.41-GCCcore-11.3.0.eb from this pull request?

Either would be fine (since there was nothing else in this PR), but as practice you can mark removal of files with a colon:
```
eb --update-pr 16353 :file-5.41-GCCcore-11.3.0.eb --pr-commit-msg="Drop old 5.41 version"
```
if done in a single step, git is also smart enough to figure out if there was a file being moved
```
eb --update-pr 123456 :some_old_name.eb some_new_name.eb --pr-commit-msg="Renamed easyconfig"
```
(but that's to late now; but it's not that important, we never really look at the git history for easyconfigs)

(on occassion, when i've had a bunch of changes to version suffixes and such, i've also just done a new PR and closed the old PR since it was quicker and i didn't wanna look at the mess i had made)

**Comment by GrahamDB** [comment]
I really appreciate this.

Thank you!
Graham

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3171.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8d1b47e3746da2438f9f0818ac33a2f3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @GrahamDB!

**Pull request merged**